internal/db/p2p_replicator.go

```diff
@@ -90,20 +88,19 @@ func (db *db) SetReplicator(ctx context.Context, rep client.Replicator) error {
 		}
 
 	default:
-		// default to all collections (unless a collection contains a policy).
-		// TODO-ACP: default to all collections after resolving https://github.com/sourcenetwork/defradb/issues/2366
 		allCollections, err := db.GetCollections(ctx, client.CollectionFetchOptions{})
 		if err != nil {
 			return NewErrReplicatorCollections(err)
 		}
 
-		for _, col := range allCollections {
-			// Can not default to all collections if any collection has a policy.
-			// TODO-ACP: remove this check/loop after https://github.com/sourcenetwork/defradb/issues/2366
-			if col.Description().Policy.HasValue() {
-				return ErrReplicatorSomeColsHavePolicy
+		if db.acp.HasValue() && !db.acp.Value().SupportsP2P() {
```

Comment:
todo: the switch case above this also checks for a policy (unsure why there is not a todo there). It might be cleaner to combine both checks outside of the switch case

Reply:
Nice spot!  Yeah I'll pull them both out of the switch and remove one of the error types.  Thanks Keenan :)

- [ ] replicator error stuff